src/_store/root.js

```diff
@@ -24,6 +24,25 @@ addLocaleData({
 
 const emptyObject = {};
 
+
+export const accountExclusion = async (token, accts = []) => {
```

Comment:
@borisyankov i think we cannot take it binary-utils. Also i don't think leaving there would do harm, or should i take it to a diff file?